core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div ng-if="collectionSummary !== null" class="conversation-skin-back-to-collection-container">
```

Comment:
If that's the case, maybe do `ng-if="collectionSummary"` and you don't need the extra boolean.